frame/democracy/src/tests/public_proposals.rs

```diff
@@ -146,9 +148,9 @@ fn blacklisting_should_work() {
 fn runners_up_should_come_after() {
 	new_test_ext().execute_with(|| {
 		System::set_block_number(0);
-		assert_ok!(propose_set_balance(1, 2, 2));
-		assert_ok!(propose_set_balance(1, 4, 4));
-		assert_ok!(propose_set_balance(1, 3, 3));
+		assert_ok!(propose_set_balance(3, 2, 2));
+		assert_ok!(propose_set_balance(3, 4, 4));
+		assert_ok!(propose_set_balance(3, 3, 3));
 		fast_forward_to(2);
 		assert_ok!(Democracy::vote(RuntimeOrigin::signed(1), 0, aye(1)));
 		fast_forward_to(4);
```

Comment:
Missing new test verifying that we actually charged the fee to the user (a.k.a. balance after propose should be equal to balance before - fee amount)

Reply:
I added "creating_proposal_takes_fee"